easybuild/easyconfigs/p/Percolator/Percolator-3.4-foss-2019a.eb

```diff
@@ -0,0 +1,78 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Bundle'
+
+name = 'Percolator'
+version = '3.4'
+
+# homepage doesn't have correctly working https
+# stating GitHub because landing page links the homepage
+# http://percolator.ms/
+homepage = 'https://github.com/percolator/percolator'
+description = "Semi-supervised learning for peptide identification from shotgun proteomics datasets"
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
I don't think this software actaully depends on any BLAS or LAPACK, so, should be fine to drop it to `gompi` as far as I can see.

Reply:
Thanks :) made the changes